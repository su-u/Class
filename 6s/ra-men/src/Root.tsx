import React from 'react';

export default () => (
    <>
        <div data-test='div1'>aoaaooaaaaaea</div>
        <div>uuuull</div>
        <h2>a2aaaa</h2>
    </>
);
