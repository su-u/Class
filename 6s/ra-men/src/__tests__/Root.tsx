import React from 'react'
import { shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import Root from '@/Root';
import { sel } from '../Utility';

describe('Root', () => {
    test('SnapShot', () => {
        const root = renderer.create(
            <Root/>
        ).toJSON();
        expect(root).toMatchSnapshot();
    });
    test('div1', () => {
        const wrapper = shallow(<Root/>);
        expect(wrapper.find(sel('div1')).text()).toEqual('aoaaooaaaaaea');
    });
    test('h2', () => {
        const wrapper = shallow(<Root/>);
        expect(wrapper.find('h2').text()).toEqual('a2aaaa');
    });
});
