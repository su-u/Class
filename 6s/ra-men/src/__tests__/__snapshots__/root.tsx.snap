// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root SnapShot 1`] = `
Array [
  <div
    data-test="div1"
  >
    aoaaooaaaaaea
  </div>,
  <div>
    uuuull
  </div>,
  <h2>
    a2aaaa
  </h2>,
]
`;
